karrot/groups/models.py

```diff
@@ -78,13 +79,15 @@ class Group(BaseModel, LocationModel, ConversationMixin, DirtyFieldsMixin):
     public_description = models.TextField(blank=True)
     application_questions = models.TextField(blank=True)
     status = models.CharField(
-        default=GroupStatus.ACTIVE.value,
-        choices=[(status.value, status.value) for status in GroupStatus],
+        # default can be overwritten by local_settings.py GroupManager
+        default=settings.GROUP_STATUS_DEFAULT.value,
+        choices=[(status.value, status.value) for status in themes.GroupStatus],
         max_length=100,
     )
     theme = models.TextField(
-        default=GroupTheme.FOODSAVING.value,
-        choices=[(theme.value, theme.value) for theme in GroupTheme],
+        # default can be overwritten by local_settings.py GroupManager
+        default=settings.GROUP_THEME_DEFAULT.value,
```

Comment:
Why do you want to remove choices? It would be necessary if we'll create a dialog so people can actually create differently themed groups. (see bike kitchen issues)

Reply:
I mean to remove `default=settings.GROUP_THEME_DEFAULT.value`, as this default is already provided by the new manager.